public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.test.tsx

```diff
@@ -21,7 +21,7 @@ const setup = () => {
       withCredentials: false,
       isDefault: false,
       jsonData: {
-        subscriptionId: '',
+        subscriptionId: '44693801-6ee6-49de-9b2d-9106972f9572',
```

Comment:
Is this a real ID?  if so, we should put in some dummy rather than pointing the world to our test instances :)